VideoLocker/src/main/java/org/edx/mobile/view/LoginActivity.java

```diff
@@ -353,6 +354,40 @@ public void showNoNetworkDialog() {
         NoNetworkFragment.show(getSupportFragmentManager(), "dialog");
     }
 
+    private void disableAllClicks() {
+        loginButtonLayout.setEnabled(false);
+
+        ImageView imgFacebook=(ImageView)findViewById(R.id.img_facebook);
```

Comment:
This would be better as one method that takes a boolean indicating if things should be enabled or not. That way it's not possible for the enable and disable methods to get out of sync.